content/en/agent/logs/proxy.md

```diff
@@ -15,8 +15,11 @@ further_reading:
 
 Log collection requires the Datadog Agent v6.0+. Older versions of the Agent do not include the `log collection` interface.
 
-By default, Datadog transports logs over TCP/SSL. Hence, there is a different [set of proxy settings][1] than other data types that the Datadog Agent forwards in HTTPS.
-Configure the Agent to send logs in HTTPS using the same set of proxy settings as other data types.
+As of Agent v6.14/v7.14, Datadog recommends the use of and enforcing HTTPS transport (see [agent Transport for Logs][2]). 
```

Comment:
I would recommend to have some **Bold** around HTTPS to insist on it.

Reply:
done